clab/exec/exec.go

```diff
@@ -101,7 +101,18 @@ func (e *ExecCmd) GetCmdString() string {
 }
 
 func (e *ExecResult) String() string {
-	return fmt.Sprintf("Cmd: %s\nReturnCode: %d\nStdOut:\n%s\nStdErr:\n%s\n", e.GetCmdString(), e.ReturnCode, e.Stdout, e.Stderr)
+	var s strings.Builder
+
+	s.WriteString(fmt.Sprintf("Cmd: %s\nReturnCode: %d", e.GetCmdString(), e.ReturnCode))
+
+	if e.Stdout != "" {
+		s.WriteString(fmt.Sprintf("\nStdout: %s", e.Stdout))
+	}
+	if e.Stderr != "" {
+		s.WriteString(fmt.Sprintf("\nStderr: %s", e.Stderr))
+	}
+
+	return s.String()
```

Comment:
@steiler I think I am going mad lol
For some reason the logger eats up the first character when printing it the `Stdout:` here.
I am curious if you can crack this mystery:

```
DEBU[0002] mgmt_server status check failed, output: 
Cmd: sr_cli -d info from state system app-management application mgmt_server state | grep running
ReturnCode: 126
 tdout: OCI runtime exec failed: exec failed: unable to start container process: exec: "sr_cli": executable file not found in $PATH: unknown
```

Note, the `tdout` string.

Reply:
for this log message to appear, change the command here from sr_cli to some garbage
https://github.com/srl-labs/containerlab/blob/ada8a8d26a07bd12ec1542610c8e37b9aa9ae149/nodes/srl/srl.go#L109